contrib/packaging_python/mk_clean_wheel.sh

```diff
@@ -40,7 +40,8 @@ source ./$ENVDIR/bin/activate
 
 # install packages
 pip install -q --upgrade pip setuptools auditwheel
-pip install -q Cython pytest tomli scikit-build flex pyparsing 
+pip install -r ../contrib/requirements_build.txt
+pip install -r ../contrib/requirements_python_dev.txt
```

Comment:
```suggestion
pip install -r contrib/requirements_build.txt
pip install -r contrib/requirements_python_dev.txt
```

I think this is called from the base directory, since `./configure.sh` is called further below without changing the directory.

Reply:
I confirm this change is required.

Reply:
Ah, I'd missed the `pushd` further down, which is why some stuff did have `../contrib`.

I've fixed this!